app/src/main/java/cz/martykan/forecastie/Broadcaster.java

```diff
@@ -0,0 +1,123 @@
+package cz.martykan.forecastie;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import cz.martykan.forecastie.models.Weather;
+import cz.martykan.forecastie.utils.UnitConvertor;
+import cz.martykan.forecastie.weatherapi.WeatherStorage;
+import nodomain.freeyourgadget.gadgetbridge.model.WeatherSpec;
+
+public class Broadcaster {
+    public static void sendWeather(Context context) {
+        SharedPreferences sp = PreferenceManager.getDefaultSharedPreferences(context);
+        boolean run = sp.getBoolean("sendToGadgetbridge", false);
+
+        if(run) {
+            WeatherStorage storage = new WeatherStorage(context);
+
+            Intent intent = new Intent();
+            intent.setAction("de.kaffeemitkoffein.broadcast.WEATHERDATA");
+            intent.setPackage("nodomain.freeyourgadget.gadgetbridge");
```

Comment:
Wondering if this should be configurable, since there are a few different flavors now (eg. nightly, bangle.js).

Reply:
That makes sense also for future compatibility.